python/tvm/testing.py

```diff
@@ -1199,18 +1238,15 @@ def wrapper(*args, **kwargs):
             except KeyError:
                 cached_value = cache[cache_key] = func(*args, **kwargs)
 
-            try:
-                yield copy.deepcopy(cached_value)
-            except TypeError as e:
-                rfc_url = (
-                    "https://github.com/apache/tvm-rfcs/blob/main/rfcs/"
-                    "0007-parametrized-unit-tests.md#unresolved-questions"
-                )
-                message = (
-                    "TVM caching of fixtures can only be used on serializable data types, not {}.\n"
-                    "Please see {} for details/discussion."
-                ).format(type(cached_value), rfc_url)
-                raise TypeError(message) from e
+            yield copy.deepcopy(
+                cached_value,
+                _DeepCopyAllowedClasses(
+                    # *args should be a list of classes that are known
+                    # to be safe to copy using copy.deepcopy, but do
```

Comment:
that are safe to copy with copy.deepcopy

Reply:
Agreed as above.